docs/requirements.txt

```
@@ -0,0 +1 @@
+sphinx==3.0.0
```

How about allowing v3.0.0 or later?
```suggestion
sphinx>=3.0.0
```

---

That makes sense, thanks.